server/middlewares/authorizeMiddleware.js

```diff
@@ -0,0 +1,54 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+class Middleware {
+  static clientData(request, response, next) {
+    try {
+      const token = request.headers.authorization.split(' ')[1];
+      const decoded = jwt.verify(token, process.env.SECRET);
+      request.userData = decoded;
+      next();
+    } catch (error) {
+      if (error.message === 'invalid token') {
```

Comment:
this error doesn't belong in the catch block, the catch block catch server errors